monitoring/angle-monitoring/src/test/java/com/farao_community/farao/monitoring/angle_monitoring/AngleMonitoringTest.java

```diff
@@ -146,20 +146,20 @@ public void testPreventiveStateOnly() {
         setUpCracFactory("network.xiidm");
```

Comment:
change mockPreventiveState thresholds to keep insecure state : the idea is that the situation cannot be secured